content/source/docs/extend/plugin-sdk.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "extend"
+page_title: "Terraform Plugin SDK"
+sidebar_current: "docs-extend-0.12-compatibility"
+description: |-
+  Official standalone SDK for Terraform plugin development
+---
+
+As of September 2019, Terraform provider developers importing the Go module `github.com/hashicorp/terraform`, known as Terraform Core, should switch to `github.com/hashicorp/terraform-plugin-sdk`, the Terraform Plugin SDK, instead. 
+
+# Why a separate module?
+
+While the `helper` package in Terraform Core has served us well, in order for provider development to evolve, the SDK needed to break out into its own repository. Terraform Core's versioning has been oriented towards practitioners. With the "unofficial" SDK existing in the core repository, the SDK becomes tied to Core releases and cannot follow semantic versioning. The new standalone SDK [github.com/hashicorp/terraform-plugin-sdk](https://github.com/hashicorp/terraform-plugin-sdk) follows sematic versioning starting with v1.0.0.
```

Comment:
Nitpick - I'd write "`helper/*` and other packages" as it's not just one package and `helper` isn't even importable package.